### PR TITLE
Creating Geometry Tests

### PR DESCRIPTION
**Pull request by puravdatta-sudo**
# Pull Requests

Please note that we accept pull requests from anyone, but that does not mean it will be merged.

## What kind of change does this PR introduce?
* Feature

## Did this PR introduce a breaking change?
* No

### CODE REVIEW

File: core/src/test/java/com/arcrobotics/ftclib/geometry/Pose2dTest.java [hunk 1]
```diff
@@ -5,56 +5,64 @@
 import static org.junit.jupiter.api.Assertions.*;
 
 class Pose2dTest {
+    private static final double kEpsilon = 1E-9;
     Pose2d base = new Pose2d(new Translation2d(0,0),new Rotation2d(0));
 
     @Test
-    void testPlus() {
```
Comment (JIceberg) [inline core/src/test/java/com/arcrobotics/ftclib/geometry/Pose2dTest.java:11]: Why was this test removed?

File: core/src/test/java/com/arcrobotics/ftclib/geometry/Transform2dTest.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package com.arcrobotics.ftclib.geometry;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class Transform2dTest {
```
Comment (JIceberg) [inline core/src/test/java/com/arcrobotics/ftclib/geometry/Transform2dTest.java:8]: Only one test?

**Review by JIceberg — CHANGES_REQUESTED** [review]
Add more tests to Transform2dTest and add the plusTest back to the Pose2dTest

File: core/src/test/java/com/arcrobotics/ftclib/geometry/Transform2dTest.java (hunk 2 above)
Comment (puravdatta-sudo) [reply core/src/test/java/com/arcrobotics/ftclib/geometry/Transform2dTest.java:8]: That's the only thing present in wpilib testing, wasn't sure if we needed more testing than they do.

Comment (JIceberg) [reply core/src/test/java/com/arcrobotics/ftclib/geometry/Transform2dTest.java:8]: Just go ahead and add some more tests for the other methods. Quick ones will work fine.

Comment (puravdatta-sudo) [reply core/src/test/java/com/arcrobotics/ftclib/geometry/Transform2dTest.java:8]: Will do.

**Review by JIceberg — APPROVED** [review]

**Pull request merged**